code/users/create-tap.js

```diff
@@ -9,8 +10,8 @@ tap.beforeEach(async () => {
 
 tap.test("create user", test => {
   const user = {
-    id: "f47ac10b-58cc-4372-a567-0e02b2c3d479",
-    name: "Pat Patterson"
+    id: uuid(),
```

Comment:
curious why you chose to require a client to supple the UUID vs the system itself?